Dockerfile

```diff
@@ -416,8 +417,13 @@ RUN mkdir -p $BAZEL_HOME/bin \
     fi \
     && chmod a+x $BAZEL_HOME/bin/bazel
 
+COPY --from=gobuild /opt/go /opt/go
+
+RUN $GOBIN/go install github.com/bazelbuild/buildtools/buildozer@latest && chmod a+x $GOBIN/buildozer
```

Comment:
Shouldn't we also make the Buildozer version configurable like we do for Bazel(isk) itself?

Reply:
I am not sure this tool as a version on its own.

For instance I get that in the container:
```
ort@f44d2997d3c0:~$ buildozer --version
buildozer version: redacted 
buildozer scm revision: redacted 
```

Reply:
Hmm, I missed that "buildozer" actually comes from a sub-directory of the "buildtools" repository. Also, https://github.com/bazelbuild/buildtools releases do not seem to be strictly aligned with https://github.com/bazelbuild/bazel releases. So, let's indeed keep it at "latest" then...